src/components/ux-flex-video/ux-flex-video.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="ux-flex-video">
+	{{#if .isDataModel}}
+	<div class="flex-video {{#class}}{{class}}{{/}}">
```

Comment:
The if around class is redundant, since the comp is isolated. Without the wrapping statement the output would be the same